src/js/media-controller.js

```diff
@@ -556,6 +556,11 @@ class MediaController extends MediaContainer {
 
   mediaSetCallback(media) {
     super.mediaSetCallback(media);
+
+    if (!media.hasAttribute('tabindex')) {
```

Comment:
this is a bit unusual, but I think alternatives for fixing #309 are probably a lot worse.